include/rocksdb/options.h

```diff
@@ -1063,6 +1063,14 @@ struct ReadOptions {
   // Default: false
   bool ignore_range_deletions;
 
+  // A callback to determine whether relevant keys for this scan exist in a
+  // given table based on the table's properties. The callback is passed the
```

Comment:
`is passed the` seems to be a grammar error.

Reply:
That does't make sense, we just make it consistent with upstream change.